doc/gettingStarted/Applications.md

```diff
@@ -0,0 +1,102 @@
+- Press the Flag button (bottom left) to add a lap.
+- The stopwatch will not yet contimue counting time while the app is closed.
+
+### Alarm
+![Alarm UI](/doc/gettingStarted/AppsScreenshots/alarm.png)
+- Ajust the time with the time picker.
+- Press the Info button in the top middle to see time remaning.
+- Use the toggle in the bottom left to turn the alarm on/off.
+- Use the button in the bottom right to change the alarm days.
+   - You can pick from once, daily, or Monday-Friday.
+InfiniPaint
+### Timer
+![Timer UI](/doc/gettingStarted/AppsScreenshots/timer.png)
+- Ajust how long the timer should go for with the time picker.
+- Press the Start button at the bottom to start/stop the timer.
+
+### Steps
+![Steps UI](/doc/gettingStarted/AppsScreenshots/steps.png)
+- The total count of steps for the current display will show in the middle of the screen.
+- The Reset button in the bottom middle resets the Trip counter. (Total of all steps taken.)
+- The progress circle shows the percentage of your daily goal completed.
+
+### Heartrate
+![Heartrate UI](/doc/gettingStarted/AppsScreenshots/Heartrate.png)
+- Press Start to start measuring your heartrate.
+    - It may take a bit to get the first measurement.
+    - The monitoring currently does not happen while your screen is off, however the screen will not go to sleep while the heartrate app is open.
+
+### Music
+![Music UI](/doc/gettingStarted/AppsScreenshots/Music.png)
+- This app shows currently playing music.
+    - Please note that this app is not very useful without a device connected.
+- Press the button in the center to play/pause, and the buttons on the left and right to go to the previous and next tracks, respectively.
+- Swipe up to get to volume controls.
+
+### InfiniPaint
+![InfiniPaint UI](/doc/gettingStarted/AppsScreenshots/InfiniPaint.png)
+- This app does not allow you to swipe from the top to exit, use the side button instead.
+- Draw on the screen to add lines.
+- Hold down in one spot to change paint colors.
+
+### Pong
+![Pong UI](/doc/gettingStarted/AppsScreenshots/Pong.png)
+- This app does not allow you to swipe from the top to exit, use the side button instead.
+- Drag your finger to move the paddle.
+- Goal: Don't let the ball go off the left side of the screen.
+
+### 2048
+![2048 UI](/doc/gettingStarted/AppsScreenshots/2048.png)
+- This app does not allow you to swipe from the top to exit, use the side button instead.
+- Play a game of 2048.
+- Swipe up, down, left, or right tomove the tiles.
+- When two tiles with the same number run into each other, they will add together.
+- Goal: Don't let the screen fill up with tiles, and get to the 2048 tile to win.
+
+### Dice
+![Dice UI](/doc/gettingStarted/AppsScreenshots/Dice.png)
+- This app is not in InfiniTime 1.14.
+- Ajust the count to change the number of dice.
+- Ajust the sides to change the number of sides.
+- Press the button at the bottom to roll the die.
```

Comment:
I would either word it...
```suggestion
- Press the button at the bottom to roll.
```
or...
```suggestion
- Press the button at the bottom to roll the die/dice.
```
...since you can change the number of dice, as described by you in line 82